src/components/footer/Footer.js

```diff
@@ -172,27 +172,21 @@ const ThirdColumn = () => {
 const BranchLinks = () => {
   // Object holding info to generate links to materialtypes * NOTICE Keys are translated.
   const branch_links = {
-    books: { pathname: "/", query: {} },
-    articles: { pathname: "/", query: {} },
-    film: { pathname: "/", query: {} },
-    ematerials: { pathname: "/", query: {} },
-    games: { pathname: "/", query: {} },
-    music: { pathname: "/", query: {} },
-    nodes: { pathname: "/", query: {} },
+    books: "/inspiration/boeger",
```

Comment:
Kan vi ikke køre `bøger`?

Reply:
På hjælpesiderne kører vi også /hjaelp og det er for at undgå mærkelige url'er når man kopiere dem dem. Så vi kører også boeger her for at holde det ens